### PR TITLE
[8.x.x] New examples of expandable row with show-expandable-icon-function input

### PR DESCRIPTION
**Pull request by pacogomezimatia**
New examples of expandable row with show-expandable-icon-function table input.

- Sync example
- Async example

### CODE REVIEW

File: src/app/main/table/05.editor/table-editor.component.html [hunk 1]
```diff
@@ -24,9 +24,9 @@
       </o-table-column>
 
       <o-table-column attr="NUMCARDS" title="NUMCARDS" class="o-table-column-centered">
-        <o-table-cell-editor-integer (editionStarted)="editionStarted($event)" (editionCancelled)="editionCancelled($event)"
+        <o-table-cell-editor-real (editionStarted)="editionStarted($event)" (editionCancelled)="editionCancelled($event)"
```
Comment (daniel-grana) [inline src/app/main/table/05.editor/table-editor.component.html:27]: Why this change?? The correct one is integer editor isn't it?

File: src/app/main/table/14.expandableRow/expandable-function-sync/table-row-expandable-function-sync.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+})
+export class TableRowExpandableFunctionSyncComponent {
+
+  getFiles(key: string) {
+    return TableUtils.getFiles(key);
+  }
+
+  hasChilds(row: any, rowIndex: number) {
+    // Check for each row if hasChilds or not to show or not the expansion icon.
+    return row.HASCHILD ? true : false;
```
Comment (daniel-grana) [inline src/app/main/table/14.expandableRow/expandable-function-sync/table-row-expandable-function-sync.component.ts:17]: It should return an Observable, shouldn't it?

File: src/app/main/table/14.expandableRow/table-row-expandable.component.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <mat-tab-group>
-  <mat-tab label="{{ 'TABLEEXPANDABLE.BUTTON.BASIC' | oTranslate}}">
+  <!-- <mat-tab label="{{ 'TABLEEXPANDABLE.BUTTON.BASIC' | oTranslate}}">
```
Comment (daniel-grana) [inline src/app/main/table/14.expandableRow/table-row-expandable.component.html:2]: Uncomment the other examples please!!

**Review by daniel-grana — CHANGES_REQUESTED** [review]

**Review by gonzalo-martinez-imt — APPROVED** [review]

**Review by daniel-grana — APPROVED** [review]
